media-player.html

```diff
@@ -1,18 +1,21 @@
 <html lang='en'>
-  <head>
-    <meta charset='utf-8' />
-    <title>Sample Media Player using HTML5's Media API</title>
-    <link href='media-player.css' rel='stylesheet' />
-  </head>
+
+<head>
+  <meta charset='utf-8' />
+  <title>Sample Media Player using HTML5's Media API</title>
+  <link href='media-player.css' rel='stylesheet' />
+</head>
+
+<body>
   <div id="main"></div>
   <script src="elm.js"></script>
   <script>
-    (function() {
-      var app = Elm.Main.fullscreen();
+    (function () {
+      var app = Elm.MediaPlayer.init({ node: document.getElementById("main") });
       var video = null;
 
-      app.ports.videoEventStream.subscribe(function(event) {
-        switch(event.kind) {
+      app.ports.videoEventStream.subscribe(function (event) {
+        switch (event.kind) {
           case "setup":
             video = document.getElementById("media-video");
             console.log(video);
```

Comment:
I'm 100% on board with the "space before parenthesis" style, but we should keep it consistent. If you're going to make this change, please make it throughout the rest of the file.

Reply:
I didnt do this intentionally, my IDE does it on save. So I guess that is ensured for the file